openlibrary/plugins/openlibrary/js/carousels_partials.js

```diff
@@ -9,7 +9,7 @@ export function initCarouselsPartials() {
             url: '/partials',
             type: 'GET',
             data: {
-                workid: $('.RelatedWorksCarousel').attr('workId'),
+                workid: $('.RelatedWorksCarousel').attr('data-workId'),
```

Comment:
While this works the same, it's better in jquery to use `.data('workId').` as it has no DOM lookup and can be read from cache.